client/layout/style.scss

```diff
@@ -72,135 +115,140 @@
+	width: $sidebar-width-max;
+	overflow: hidden;
 
-	.is-section-preview & {
-		height: 100%;
+	@include breakpoint( "<960px" ) {
+		width: $sidebar-width-min;
 	}
-}
 
-.layout__content a {
-	text-decoration: none;
-}
+	@include breakpoint( "<660px" ) {
+		width: 100%;
+	}
 
-.layout {
-	.sidebar,
-	.layout__secondary .site-selector,
-	.current-site,
-	.sidebar__menu {
-		transform: translateX( 0 );
-		transition: all 0.15s cubic-bezier(0.075, 0.820, 0.165, 1.000);
+	// The editor has its own sidebar which doesnt
+	// use the secondary element.
+	.is-section-post-editor & {
+		display: none;
 	}
 }
 
-.layout {
-	.sidebar,
-	.masterbar {
-		@media print {
-			display: none;
+// Setup the site-selector element. Its default
+// position is off screen.
+.layout__secondary .site-selector {
+	position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+	pointer-events: none;
+	transform: translateX( -$sidebar-width-max );
+	pointer-events: none;
+
+	.site,
+	.all-sites {
+		.site__title,
+		.site__domain {
+			&::after {
+				@include long-content-fade( $color: $gray-lighten-30 );
+			}
 		}
 	}
+
+	@include breakpoint( "<660px" ) {
+		-webkit-overflow-scrolling: touch;
+		transform: translateX( -100% );
+	}
 }
 
-.layout.focus-sites {
+
+/*
+	Focus States
+	Sites - Site Selector for those with multiple sites
+	Sidebar - The sidebar is the current focus
+	Content - The content is the furrent focus
+*/
+
+// Adjust the content as needed when focused on the site selector
+// or sidebar, but never in the editor.
+.layout.focus-sites:not(.is-section-post-editor),
+.layout.focus-sidebar:not(.is-section-post-editor) {
 	.layout__primary {
-		opacity: 0.2;
 		pointer-events: none;
+
+		@include breakpoint( "<660px" ) {
+			transform: translateX( 100% );
+			overflow: hidden;
+			height: calc( 100% - 47px );
+		}
 	}
+}
 
+// Adjust elements when the site-selector is visible.
+.layout.focus-sites {
 	.layout__secondary .site-selector {
-		opacity: 1;
-		transform: translateX( 272px );
 		pointer-events: auto;
+		transform: translateX( 0 );
+	}
+
+	.layout__secondary .sidebar {
+		pointer-events: none;
+		transform: translateX( $sidebar-width-max );
 
 		@include breakpoint( "<660px" ) {
 			transform: translateX( 100% );
 		}
 	}
 
-	.sidebar {
-		pointer-events: none;
-	}
-
-	.current-site,
-	.sidebar__menu {
-		opacity: 0;
-		transform: translateX( 64px );
+	.layout__primary {
+		@include breakpoint( ">660px" ) {
+			opacity: 0.25;
+		}
 	}
 }
 
-.layout.focus-sidebar {
-	overflow: hidden;
-}
-
-.layout.focus-sidebar:not(.is-section-post-editor) .layout__primary {
-	//when sidebar is focused, create a z-index stacking context on primary, so elements don't bleed in mobile views.
-	transform: translate( 0, 0 );
+// Move the secondary element off screen when focused
+// on the content. This only applies to small screens.
+.layout.focus-content {
+	@include breakpoint( "<660px" ) {
+		.layout__secondary {
+			transform: translateX( -100% );
```

Comment:
Is it possible to `display: none` instead? This would save the browser some work.

Reply:
Yes, its possible — but then we lose the "sliding" transition.